test/v1/connectors/AmendPeriodSummaryConnectorSpec.scala

```diff
@@ -60,41 +62,53 @@ class AmendPeriodSummaryConnectorSpec extends ConnectorSpec {
     )
   )
 
-  class Test extends MockHttpClient with MockAppConfig {
+  val nonTysRequest = makeRequest(None)
+  val tysRequest    = makeRequest(Some("2023-24"))
 
-    val connector: AmendPeriodSummaryConnector = new AmendPeriodSummaryConnector(
-      http = mockHttpClient,
+
+  trait Test {
+    _: ConnectorTest =>
+
+    protected val connector: AmendPeriodSummaryConnector = new AmendPeriodSummaryConnector(
+      http      = mockHttpClient,
       appConfig = mockAppConfig
     )
-
-    MockAppConfig.desBaseUrl returns baseUrl
-    MockAppConfig.desToken returns "des-token"
-    MockAppConfig.desEnvironment returns "des-environment"
-    MockAppConfig.desEnvironmentHeaders returns Some(allowedDesHeaders)
   }
 
   "AmendPeriodSummaryConnector" when {
+
     "amendPeriodSummary" must {
-      "return a 204 status for a success scenario" in new Test {
+
+      "return a 204 status for a success TYS scenario" in new TysIfsTest with Test {
+
+        val outcome = Right(ResponseWrapper(correlationId, ()))
+
+        val fromDate: String = tysRequest.periodId.substring(0, 10)
+        val toDate: String   = tysRequest.periodId.substring(11, 21)
```

Comment:
Appreciate it was already doing this for non-TYS, but it'd be better if these values were hardcoded in the `url` string instead of calculated like this, because we're just copying the logic from the connector, so if the substring indices are incorrect the tests won't catch it.